esp_encrypted_img/test_apps/sdkconfig.defaults

```diff
@@ -0,0 +1,4 @@
+# This file was generated using idf.py save-defconfig. It can be edited manually.
+# Espressif IoT Development Framework (ESP-IDF) 5.4.0 Project Minimal Configuration
+#
+CONFIG_ESP_TASK_WDT_INIT=n
```

Comment:
Most of the test apps like `esp_encrypted_img`, `json_parser` may not require this additional config. I think they should work with task watchdog enabled case too.

Reply:
It doesn't impact CI, however when you run the app locally, there will be a task watchdog timeout due to a busy loop in unity_run_menu.